MultipazGettingStartedSample/composeApp/src/commonMain/kotlin/org/multipaz/getstarted/App.kt

```diff
@@ -246,7 +254,7 @@ class App {
                         privacyPolicyUrl = "https://verifier.multipaz.org/identityreaderbackend/"
                     )
                 )
-            } catch (e: TrustPointAlreadyExistsException) {
+            } catch (e: Throwable) {
```

Comment:
why? isn't only `TrustPointAlreadyExistsException` thrown here? also, catching `Throwable` is not a good practice.